### ISSUE TITLE
Pairing via Frontend - Problem

### ISSUE THREAD

**Issue by SaschaMuehl** [danielwelch/hassio-zigbee2mqtt#326]
**Description of problem:**
Hallo, could you tell me where I should place your mentoined script for the entities please. 

Must I copy it in:  
`/opt/zigbee2mqtt/data/configuration.yaml`

or in 
`/PATH_TO_YOUR_CONFIG/configuration.yaml`
?
```
> mqtt:
  broker: core-mosquitto # This will have to be your mqtt broker, Mosquitto addon is recommended.
  discovery: true

input_boolean:
  zigbee_permit_join:
    name: Allow devices to join
    initial: off
    icon: mdi:cellphone-wireless

timer:
  zigbee_permit_join:
    name: Time remaining
    duration: 600 # Updated this to the number of seconds you wish

sensor:
  - platform: mqtt
    name: Bridge state
    state_topic: "zigbee2mqtt/bridge/state"
    icon: mdi:router-wireless

automation:
  - id: enable_zigbee_join
    alias: Enable Zigbee joining
    trigger:
      platform: state
      entity_id: input_boolean.zigbee_permit_join
      to: 'on'
    action:
    - service: mqtt.publish
      data:
        topic: zigbee2mqtt/bridge/config/permit_join
        payload: 'true'
    - service: timer.start
      data:
        entity_id: timer.zigbee_permit_join
  - id: disable_zigbee_join
    alias: Disable Zigbee joining
    trigger:
    - entity_id: input_boolean.zigbee_permit_join
      platform: state
      to: 'off'
    action:
    - data:
        payload: 'false'
        topic: zigbee2mqtt/bridge/config/permit_join
      service: mqtt.publish
    - data:
        entity_id: timer.zigbee_permit_join
      service: timer.cancel
  - id: disable_zigbee_join_timer
    alias: Disable Zigbee joining by timer
    trigger:
    - platform: event
      event_type: timer.finished
      event_data:
        entity_id: timer.zigbee_permit_join
    action:
    - service: mqtt.publish
      data:
        topic: zigbee2mqtt/bridge/config/permit_join
        payload: 'false'
    - service: input_boolean.turn_off
      data:
        entity_id: input_boolean.zigbee_permit_join
```

![grafik](https://user-images.githubusercontent.com/62723171/79953485-36e4d900-847c-11ea-99d5-fe48055078ad.png)

![grafik](https://user-images.githubusercontent.com/62723171/79953586-61cf2d00-847c-11ea-8f61-dfff8d62866b.png)

Thank you for your help. I'm quite new in hassio.

**Comment by ServerDotBiz** [comment]
I am new to hassio / home-automation aswell and this step isn't quite clear to me either.

Where do you add this configuration?

**Comment by ServerDotBiz** [comment]
After some more investigating I ended up installing the "File editor" addon.

Once you have done this, simply edit your maijn "configuration.yaml" file and add your YAML.

The next step is to added the Lovelace YAML to the frontend using the "raw configuration editor", top right menu.

I guess the documentation assumes some experience with Home Assistant, I started a fresh install this morning. So far I am impressed with all the functionality!
